training_toolbox/setup.py

```diff
@@ -0,0 +1,15 @@
+import os
+from setuptools import setup
+
+with open('./requirements.txt') as f:
+  required = f.read().splitlines()
+
+# Under the Travis we have to use a CPU version
+if os.environ.get('TRAVIS', None) == 'true':
+  required = [p.replace('-gpu', '') if p.startswith('tensorflow') else p for p in required]
+
+setup(
+    name='Training toolbox',
+    version='1.0.0.dev0',
```

Comment:
Current version is 0.1.0: https://github.com/opencv/training_toolbox_tensorflow/blob/develop/CHANGELOG.md